src/test/java/MapTest.java

```diff
@@ -0,0 +1,13 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MapTest {
+    @Test
+    public void checkMapTest() {
+        MapExample.characterCount("test");
+        assertEquals(Map.of('t', 2, 'e', 1, 's', 1), MapExample.characterCount("test"), "Letters and their number are considered correct");
```

Comment:
Please reformat this code (using Code - Reformat code context menu, if it doesn't do anything - do it manually) - it is hard to read such a long line:
<img width="1478" alt="image" src="https://user-images.githubusercontent.com/90613721/181313560-a28aca9f-3a9a-4a42-a476-4c2182d844fd.png">

Reply:
Ctrl+Alt+L didn't work, I did that manually